ansible/ci-deployment.yml

```diff
@@ -33,8 +47,26 @@
     - role: omero-web-runtime
     - role: nginx
 
+
 # Deploy Documentation prerequisites
 - hosts: ci-docs
   roles:
+    - role: virtualenv
     - role: sphinx-build
     - role: ansible
+
+  vars:
+    virtualenv_user: "{{ jenkinsuser }}"
+    virtualenv_path: /home/{{ jenkinsuser }}/virtualenv
+    virtualenv_site_packages: "yes"
+    virtualenv_pip_rpm_dependencies:
+    virtualenv_pip_dependencies:
```

Comment:
@sbesson for autogen I had to add https://github.com/openmicroscopy/infrastructure/pull/129. Does it mean is no longer the case? We don't need venv at all?